app/globalReducers/app.js

```diff
@@ -31,6 +31,8 @@ const initialState = fromJS({
   pathfinderUrl: '/',
   moicUrl: '/',
   licencesUrl: '/',
+  dataComplianceUrl: '/',
```

Comment:
Is this needed? I saw you removed it from above. Not a biggie

Reply:
Thanks, removed